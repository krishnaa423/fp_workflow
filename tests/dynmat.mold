[Molden Format]
[FREQ]
    0.00
    0.00
    0.00
  584.40
  584.40
  584.40
[FR-COORD]
   Si          0.00000        0.00000        0.00000
   Si          2.56530        2.56530        2.56530
[FR-NORM-COORD]
 vibration     1
   0.13370  -0.17136   0.67287
   0.13370  -0.17136   0.67287
 vibration     2
  -0.41975   0.52590   0.21734
  -0.41975   0.52590   0.21734
 vibration     3
  -0.55311  -0.44053  -0.00229
  -0.55311  -0.44053  -0.00229
 vibration     4
   0.00000   0.00000  -0.70711
   0.00000  -0.00000   0.70711
 vibration     5
  -0.67082  -0.22361  -0.00000
   0.67082   0.22361  -0.00000
 vibration     6
   0.22361  -0.67082   0.00000
  -0.22361   0.67082   0.00000
