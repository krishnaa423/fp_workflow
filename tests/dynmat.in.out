
     Program DYNMAT v.7.2 starts on 17Jul2024 at  5:10:21 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     28148 MiB available memory on the printing compute node when the environment starts
 

     Reading Dynamical Matrix from file struct.dyn1
     ...Force constants read
     ...epsilon and Z* read
     Acoustic Sum Rule: || Z*(ASR) - Z*(orig)|| =    1.327391E+01
     Acoustic Sum Rule: ||dyn(ASR) - dyn(orig)||=    5.242571E-04
     A direction for q was not specified:TO-LO splitting will be absent

     Polarizability (A^3 units)
     multiply by 0.050198 for Clausius-Mossotti correction
       180.799223    0.000000   -0.000000
         0.000000  180.799223   -0.000000
        -0.000000   -0.000000  180.799223

     IR activities are in (D/A)^2/amu units

# mode   [cm-1]    [THz]      IR
    1     -0.00   -0.0000    0.0000
    2      0.00    0.0000    0.0000
    3      0.00    0.0000    0.0000
    4    584.40   17.5199    0.0000
    5    584.40   17.5199    0.0000
    6    584.40   17.5199    0.0000
 
     DYNMAT       :      0.02s CPU      0.01s WALL

 
   This run was terminated on:   5:10:21  17Jul2024            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
